presto-tpch/src/test/java/com/facebook/presto/tpch/statistics/ColumnStatisticsRecorder.java

```diff
@@ -13,12 +13,22 @@
  */
 package com.facebook.presto.tpch.statistics;
 
+import io.airlift.tpch.TpchColumnType;
```

Comment:
Consider moving the statistics recorder to the `presto-tests`, so you can reuse it in both, `TPC-H` and `TPC-DS`

Reply:
I don't think they are identical. Anyway, let's consider this outside for this PR.